phpcs.xml.dist

```diff
@@ -2,7 +2,7 @@
 <ruleset>
     <arg name="basepath" value="."/>
     <arg name="extensions" value="php"/>
-    <arg name="parallel" value="80"/>
+    <arg name="parallel" value="8"/>
```

Comment:
Why this change?

Reply:
It seems to me that starting 80 processes is a typo copied from somewhere else. Maybe 8 processes was the original intention.

Reply:
Looks like phpcs itself is run with a similar value: https://github.com/squizlabs/PHP_CodeSniffer/commit/a57331c6c033ef3c947415898b9921a4237289cf

Reply:
Happy to revert. Just seems odd.

Reply:
Did you see my edit?

Reply:
Interesting. I'll research about `parallel` and `phpcs` to see if this change is right because then we'd need to propagate it to the other Doctrine projects. Thanks for taking a look into that :)

Reply:
shall i revert? normally tools like this set the number of parallel process to the number of available cores, or something like that. 80 just seems wild. not sure why PHPCS is using 75 either.

Reply:
> normally tools like this set the number of parallel process to the number of available cores

That makes sense to me if your process is CPU bound, but if the process spends 90 percent of its time actually being IO bound, it would make sense to me to multiply the number of cores by some factor, so that the scheduler can give the CPU to a process that is no longer waiting for IO. But again, maybe what I'm saying does not make sense. Can you try locally and tell us what figure results in the fastest processing time for you?

Reply:
I've reverted so we can get this merged but:
```
➜  dbal git:(phpcs-github-actions) ✗ time vendor/bin/phpcs --no-cache --parallel=8
........ 8 / 8 (100%)


Time: 12.85 secs; Memory: 8MB

vendor/bin/phpcs --no-cache --parallel=8  51.49s user 3.37s system 423% cpu 12.945 total
➜  dbal git:(phpcs-github-actions) ✗ time vendor/bin/phpcs --no-cache --parallel=1
............................................................  60 / 456 (13%)
............................................................ 120 / 456 (26%)
............................................................ 180 / 456 (39%)
............................................................ 240 / 456 (53%)
............................................................ 300 / 456 (66%)
............................................................ 360 / 456 (79%)
............................................................ 420 / 456 (92%)
....................................                         456 / 456 (100%)


Time: 28.9 secs; Memory: 34MB

vendor/bin/phpcs --no-cache --parallel=1  28.56s user 0.23s system 99% cpu 28.986 total
➜  dbal git:(phpcs-github-actions) ✗ time vendor/bin/phpcs --no-cache --parallel=8
........ 8 / 8 (100%)


Time: 12.7 secs; Memory: 8MB

vendor/bin/phpcs --no-cache --parallel=8  49.23s user 3.21s system 410% cpu 12.780 total
➜  dbal git:(phpcs-github-actions) ✗ time vendor/bin/phpcs --no-cache --parallel=80
............................................................ 60 / 76 (79%)
................                                             76 / 76 (100%)


Time: 9.2 secs; Memory: 8MB

vendor/bin/phpcs --no-cache --parallel=80  47.74s user 2.92s system 546% cpu 9.264 total
```

Reply:
Indeed almost no gain, so this is probably more CPU-bound nowadays.